scripts/world_bank/health_nutrition_population/population.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+"""world_bank_population.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1f5j8kacwOd_dZqxoO_XCtwuk2jck67bz
+    
+final csv file:
+https://drive.google.com/file/d/1YY8ewSuZD-wx9qJNdvjmz9MB0qJxP-7L/view?usp=sharing&resourcekey=0-ncNblwybOhMGrC2IFPDf_w
+"""
+
+import requests
+import pandas as pd
+
+AGES = 26
+
+series  = [f"SP.POP.AG{age:02d}.{gender}.IN" for age in range(AGES)
+           for gender in ['FE', 'MA']]
+
+def get_df(serieses):
+    '''
+    gets df by iteratively running code for each country + series
+    '''
+    df2 = pd.DataFrame
+    df2 = pd.DataFrame(columns = ['Series Name', 'Series Code', 'Country Code', 'Country',
+                                  'Year', 'Value'])
+    current_series = "&".join(serieses)
+    MAX_PER_PAGE = 32767
+    url = f"https://api.worldbank.org/v2/country/all/indicator/{current_series}?"
+    url = url + f"format=JSON&per_page={MAX_PER_PAGE}"
+    print(url)
+
+    response = requests.get(url)
+    response = response.json()
+    for counter in range(len(response[1])):
+        val = response[1][counter]['value']
+        if val is not None:
+            row = {"Series Name" : response[1][counter]['indicator']['value'],
+                "Series Code" : response[1][counter]['indicator']['id'],
+                "Country Code" : response[1][counter]['countryiso3code'],
+                "Country" : response[1][counter]['country']['value'],
+                "Year" : response[1][counter]['date'],
+                'Value' : response[1][counter]['value']
+
+                }
+            df2.loc[len(df2)] = row
+    return df2
+
+def get_mcf(series_lst):
+    '''
+    gets mcf by splitting description and using fstrings
+    '''
+    nodes = []
+    #don't  use df; automate
+    with open("World_bank_hnp_population.mcf", 'w', encoding = 'utf-8') as file:
+        print(end = '')
```

Comment:
what is this print statement for, since it's printing an empty string, with no newline after.